src/media/css/reviews.styl

```diff
@@ -18,6 +18,11 @@ select[name=rating] {
         position: absolute;
         right: 0;
         top: 1px;
+
+        +rtl() {
+            position: static;
+            margin: 3px 5px 0 0;
```

Comment:
The RTL mocks asked for next to the user id:

![](http://i.imgur.com/I6dRckF.png)